src/components/common/Tree/Tree.js

```diff
@@ -52,12 +54,12 @@ const DynamicTreeView = ({
         nodeId={rootId}
         label={rootLabel}
       >
-        {itemsFiltered.map(({ id: itemId }) => (
+        {itemsFiltered.map((item) => (
           <CustomTreeItem
-            key={itemId}
-            itemId={itemId}
+            key={item.id}
             expandedItems={expandedItems}
             selectedId={selectedId}
+            itemProp={item}
```

Comment:
I think it makes sense to call it `item` only.

Reply:
The problem is that then it is used recursively in the Item component, where the item variable is the data from the useItem hook